### ISSUE TITLE
Icon buttons are not reachable with arrow key navigation

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4628]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 2.1.1 Keyboard

**Test Environment:**
- Test URL: https://clarity.design/documentation/buttons
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 20.1

**Steps to reproduce:**
1. Open the test url.
2. Go to the icon buttons section and try to navigate through list of horizontally placed icon buttons using down arrow keys.

**Expected result:**
Screen reader user should be able to navigate through the list of icon buttons using arrow keys or tab key navigation.

**Actual result:**
Screen reader user is able to navigate through icon buttons using tab key navigation only. Even tried  switching between NVDA browse/focus mode, still navigation happens with tab key only.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
